Resources/Changelog/GabyChangelog.yml

```
@@ -222,4 +222,10 @@ Entries:
     - type: Tweak
       message: Altera sprites dos alertas, IPCs, algemas e cartas.
   id: 30
-  time: '2025-02-02T06:17:26.0000000+00:00'
\ No newline at end of file
+  time: '2025-02-02T06:17:26.0000000+00:00'
+  author: Dreykor (arguemeu)
```

YAMLlint (1.35.1)

[error] duplication of key "author" in mapping

(key-duplicates)